Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -1145,20 +1660,42 @@ void MolDraw2D::extractAtomCoords(const ROMol &mol, int confId,
     bbox_[1].x = bbox_[1].y = -1 * numeric_limits<double>::max();
   }
   const RDGeom::POINT3D_VECT &locs = mol.getConformer(confId).getPositions();
-  ROMol::VERTEX_ITER this_at, end_at;
-  boost::tie(this_at, end_at) = mol.getVertices();
-  while (this_at != end_at) {
-    int this_idx = mol[*this_at]->getIdx();
+
+  // the transformation rotates anti-clockwise, as is conventional, but
+  // probably not what our user expects.
+  double rot = - drawOptions().rotate * M_PI / 180.0;
+  // assuming that if drawOptions().rotate is set to 0.0, rot will be
+  // exactly 0.0 without worrying about floating point number dust.  Does
+  // anyone know if this is true?  It's not the end of the world if not,
+  // as it's just an extra largely pointless rotation.
+  // Floating point numbers are like piles of sand; every time you move
+  // them around, you lose a little sand and pick up a little dirt.
+  // — Brian Kernighan and P.J. Plauger
+  // Nothing brings fear to my heart more than a floating point number.
+  // — Gerald Jay Sussman
+  // Some developers, when encountering a problem, say: “I know, I’ll
+  // use floating-point numbers !”   Now, they have 1.9999999997 problems.
+  // — unknown
+  double costh = rot == 0.0 ? 0.0 : cos(rot);
```

Comment:
You could do the transformation here using this class and avoid duplicating some code:
https://github.com/rdkit/rdkit/blob/master/Code/Geometry/Transform2D.h#L21